equinix/resource_fabric_connection.go

```diff
@@ -50,7 +50,7 @@ func resourceFabricConnectionCreate(ctx context.Context, d *schema.ResourceData,
 	projectReq := d.Get("project").(*schema.Set).List()
 	project := projectToFabric(projectReq)
 	additionalInfo := d.Get("additional_info").([]interface{})
-	additionalinfo := additionalInfoToFabric(additionalInfo)
+	additionalinfoGoToTerraState := additionalInfoToFabric(additionalInfo)
```

Comment:
This is actually `additionalInfoTerraConfigToGo`. The i should be capitalized in info and the data flow is Terraform Config -> Go Struct.

However, the variable shouldn't be the one naming the operation, the method should be. Everywhere we have `ToFabric` should become `TerraToGo` in the methods themselves. 

I would name the variable on line 52 `additionalInfoTerraConfig` and make the variable on line 53 be `additionalInfo`.

If you want to change the method name with this PR you can but not necessary.

Reply:
I have changed both the variable names and the method name.